elections/2020/SC/csantanapr.md

```diff
@@ -0,0 +1,23 @@
+# Carlos Santana Candidate Profile
+
+## Company Affiliation:
+
+IBM
+
+## Contributions to Knative (list):
+
+* Involved with project since 2018
+* Initial contributions in design and architecture to eventing
+* Speaker in conferences, meetups and online events for Knative (ie ibm-think, kubecon, oss, apachecon, knative, ibm-developer, etc)
+* Maintain many tutorials used for presentations [knative-minikube](https://github.com/csantanapr/knative-minikube), [knative-kind](https://github.com/csantanapr/knative-kind), [knative-tekton](https://github.com/csantanapr/knative-tekton), [knative-pi](https://github.com/csantanapr/knative-pi)
+* Contributions to docs and samples
+* Active awareness of knative resources and activities in twitter and slack.
+* Co-started weekly Friday Office/Hacky Hours
+
+## Why You're Running for SC (max 300 words)
+
+I want to contribute my time, energy, and passion on growing a healthy community for Kantive.
```

Comment:
nit: Kantive -> Knative :) Marks fave typo :)

Reply:
Thanks for catching the typo, sent PR #369